api/src/modules/indicators/indicator.entity.ts

```diff
@@ -49,8 +49,9 @@ export class Indicator extends BaseEntity {
   @Column({ type: 'text', nullable: true, unique: true })
   shortName?: string;
 
+  // TODO: Make this non-nullable and update tests
   @Column({ type: 'text', nullable: true, unique: true })
```

Comment:
maybe remove nullable: true ?

Reply:
Yes it definitely should, but I'd wait to validate this as it will require to update a lot of tests, and since we rely 95% on e2e, lots not affected by these changes as well, is going to be a pain.